stdlib/LinearAlgebra/src/uniformscaling.jl

```diff
@@ -74,10 +74,12 @@ oneunit(J::UniformScaling{T}) where {T} = oneunit(UniformScaling{T})
 zero(::Type{UniformScaling{T}}) where {T} = UniformScaling(zero(T))
 zero(J::UniformScaling{T}) where {T} = zero(UniformScaling{T})
 
+isdiag(::UniformScaling) = true
 istriu(::UniformScaling) = true
 istril(::UniformScaling) = true
 issymmetric(::UniformScaling) = true
 ishermitian(J::UniformScaling) = isreal(J.λ)
+isposdef(J::UniformScaling) = J.λ > zero(J.λ)
```

Comment:
This fails when `J.λ` is complex with imaginary part 0 (since there is no comparison operator).

You can simply do `isposdef(J::UniformScaling) = isposdef(J.λ)`.